src/components/layout/AdvancedToggle/AdvancedToggle.tsx

```diff
@@ -38,6 +39,12 @@ export default class AdvancedToggle extends React.Component<IProps, IState> {
 		});
 	}
 
+	onKeyDown (event: any) {
+		if (event.key === 'Enter' || event.key === ' '){
+			this.toggleAdvanced();
+		}
+	}
+
```

Comment:
Really nice work on this component, as well! I was worried about this one but this is super slick.